### DISCUSSION TITLE
拿 PCL 做盈利的东西会不会算作侵权

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3736] [讨论]
【国服2B2T最强水晶PVP客户端？-哔哩哔哩】 https://b23.tv/53ZVvE3
@LTCatt 
这人在简介还写道 “20CNY”，结合评论得出这应当为价格

**Comment by MoYuan-CN** [comment]
就看他卖的是客户端（整合包）还是游戏副本（含 PCL 启动器）了……

**Reply by allMagicNB** [reply]
应当是包含的，展示了主页文件

**Comment by LTCatt** [comment]
评价为玩 2B2T 的一般惹不起

**Reply by Griseo-bh3rd** [reply]
~前情提要：一两年前PCL2仓库曾经被某外挂作者交了个DMCA Takedown~

**Comment by WorldHim** [comment]
挂端与启动器没有太大关联吧。
而且他的主要卖点也不是主页。。。

**Comment by littlekan233** [comment]
> ~前情提要：一两年前PCL2仓库曾经被某外挂作者交了个DMCA Takedown~

草 介么离谱？？？？？？？

**Comment by wuliaodexiaoluo** [comment]
评价: 我们最好不要和 2B2T 有什么联系，因为惹不起
